### PR TITLE
Fetch dynamic CNV/ODF versions for must gather

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

An OpenShift cluster upgrade can fail due to fetching dynamic CNV/ODF versions for must-gather

### CODE REVIEW

File: benchmark_runner/common/oc/oc.py [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ def generate_odf_must_gather(self, destination_path: str = '/tmp', odf_version:
         :raises: RuntimeError if the command fails.
         """
         if not odf_version:
-            raise ValueError("ODF version must be provided")
+            odf_version = ".".join(self.get_odf_version().split(".")[:2])
```
Comment (RobertKrawitz) [inline benchmark_runner/common/oc/oc.py:1390]: If get_odf_version() fails, I presume it raises an exception?  Also, what if it returns something, but that isn't valid?

File: benchmark_runner/common/oc/oc.py [hunk 2]
```diff
@@ -1415,7 +1415,7 @@ def generate_cnv_must_gather(self, destination_path: str = '/tmp', cnv_version:
         :raises: RuntimeError if the command fails.
         """
         if not cnv_version:
-            raise ValueError("CNV version must be provided")
+            cnv_version = ".".join(self.get_cnv_version().split(".")[:2])
```
Comment (RobertKrawitz) [inline benchmark_runner/common/oc/oc.py:1418]: Same comment as above.

File: benchmark_runner/workloads/bootstorm_vm.py [hunk 3]
```diff
@@ -207,8 +207,8 @@ def _verify_vm_ssh(self):
             if self._wait_for_upgrade_version:
                 logger.info(f'Cluster is upgraded to: {self._wait_for_upgrade_version}')
             if failure:
-                self._oc.generate_cnv_must_gather(destination_path=self._run_artifacts_path, cnv_version=self._cnv_version)
-                self._oc.generate_odf_must_gather(destination_path=self._run_artifacts_path, odf_version=self._odf_version)
+                self._oc.generate_cnv_must_gather(destination_path=self._run_artifacts_path)
+                self._oc.generate_odf_must_gather(destination_path=self._run_artifacts_path)
```
Comment (RobertKrawitz) [inline benchmark_runner/workloads/bootstorm_vm.py:211]: What is the reason for this change?

Comment (ebattat) [reply benchmark_runner/workloads/bootstorm_vm.py:211]: We dont know when it will be failed before upgrade or after, so we dont know the cnv/odf version in advance

File: benchmark_runner/common/oc/oc.py (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/common/oc/oc.py:1420]: fixed

File: benchmark_runner/common/oc/oc.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/common/oc/oc.py:1390]: fixed

File: benchmark_runner/workloads/bootstorm_vm.py (hunk 3 above)
Comment (RobertKrawitz) [reply benchmark_runner/workloads/bootstorm_vm.py:211]: OK
